### ISSUE TITLE
in a2 question e ,it need a small written quenstion.

### ISSUE THREAD

**Issue by jcjview** [lrs1353281004/CS224n_learning_notes#1]
Very nice HW!

in a2 question e ,it need a small written quenstion.

>After you’ve done this, describe with one sentence why this loss function is much more efficient to compute than the naive-softmax loss.
